core/src/test/java/org/apache/calcite/test/ScannableTableTest.java

```diff
@@ -278,9 +278,10 @@ public class ScannableTableTest {
     final Table table = new BeatlesProjectableFilterableTable(buf, false);
     final String explain = "PLAN="
         + "EnumerableAggregate(group=[{0}], C=[COUNT()])\n"
-        + "  EnumerableAggregate(group=[{0, 1}])\n"
-        + "    EnumerableInterpreter\n"
-        + "      BindableTableScan(table=[[s, beatles]], filters=[[=($2, 1940)]], projects=[[2, 0]])";
+        + "  EnumerableCalc(expr#0=[{inputs}], expr#1=[1940], k=[$t1], i=[$t0])\n"
```

Comment:
I personally find it very difficult to evaluate whether these plans are correct by inspection (including the ones in XML files). Or even if they are better in any way.
On the other hand this PR is all about plan quality.
So I will have to trust you that the plans are better than they used to be before.